df_chat/tests/test_message_endpoint.py

```diff
@@ -17,15 +22,141 @@ def test_message_creation_success(self):
 
         message_endpoint = reverse("rooms-messages-list", kwargs={"room_pk": room.pk})
         self.client.credentials(HTTP_AUTHORIZATION="Bearer " + token)
-        response = self.client.post(message_endpoint, {"body": "Hi"})
+        body = "Test message 1234."
+        response = self.client.post(message_endpoint, {"body": body})
         message = response.json()
 
         room_user = user.roomuser_set.get()
-        self.assertEqual(message["body"], "Hi")
+        self.assertEqual(message["body"], body)
         self.assertTrue(message["is_me"])
         self.assertEqual(message["room_user_id"], room_user.pk)
         self.assertIsNone(message["parent_id"])
         self.assertFalse(message["is_reaction"])
+        self.assertEqual(response.status_code, http.HTTPStatus.CREATED)
+
+        return user, room
+
+    def test_filter_message_by_keyword(self):
+        """
+        Test-case for filtering messages by the `Message` model's `body` field.
+        """
+        _, room = self.test_message_creation_success()
+        messages_list_endpoint = reverse(
+            "rooms-messages-list", kwargs={"room_pk": room.pk}
+        )
```

Comment:
Please declare it as an attribute on the TestMessageEndpoint class and use it in all test methods.

Reply:
Done!